org.civicrm.bootstrapcivihr/scss/components/outside-namespace/_user-menu.scss

```diff
@@ -15,18 +15,13 @@
 }
 
 .chr_user-menu__data {
-  > .chr_user-menu__name {
-    float: left;
-  }
-
   > .chr_profile-card {
```

Comment:
Don't we add an empty line before nesting in order not to confuse with the properties?